code/modules/research/designs/mecha_designs.dm

```diff
@@ -43,6 +43,27 @@
 	build_path = /obj/item/circuitboard/mecha/odysseus/peripherals
 	category = list("Exosuit Modules")
 
+// Nkarrdem
+/datum/design/nkarrdem_main
+	name = "Exosuit Board (\"Nkarrdem\" Central Control module)"
+	desc = "Allows for the construction of a \"Nkarrdem\" Central Control module."
+	id = "nkarrdem_main"
+	req_tech = list("programming" = 3,"biotech" = 3, "engineering" = 3)
```

Comment:
inconsistent spacing, can you add a space after programming for these and the previous boards for consistency?